enterprise_access/apps/api_client/tests/test_utils.py

```diff
@@ -15,3 +21,106 @@ def __init__(self, json_data, status_code):
+            "start_date": _days_from_now(-50, DATE_FORMAT_ISO_8601),
+            "expiration_date": _days_from_now(50, DATE_FORMAT_ISO_8601),
+            "enterprise_customer_uuid": self.mock_enterprise_customer_uuid,
+            "enterprise_catalog_uuid": self.mock_enterprise_catalog_uuid,
+            "is_active": True,
+            "is_current": True,
+            "is_revocation_cap_enabled": False,
+            "days_until_expiration": _days_from_now(50, DATE_FORMAT_ISO_8601),
+            "days_until_expiration_including_renewals": _days_from_now(50, DATE_FORMAT_ISO_8601),
+            "is_locked_for_renewal_processing": False,
+            "should_auto_apply_licenses": False,
+            "created": _days_from_now(-60, DATE_FORMAT_ISO_8601)
+        }
+        self.mock_customer_agreement = {
+            "uuid": self.mock_customer_agreement_uuid,
+            "enterprise_customer_uuid": self.mock_enterprise_customer_uuid,
+            "enterprise_customer_slug": self.mock_enterprise_customer_slug,
+            "default_enterprise_catalog_uuid": self.mock_enterprise_catalog_uuid,
+            "disable_expiration_notifications": False,
+            "net_days_until_expiration": _days_from_now(50, DATE_FORMAT_ISO_8601),
+            "subscription_for_auto_applied_licenses": self.mock_subscription_plan_uuid,
+            "available_subscription_catalogs": [
+                self.mock_enterprise_catalog_uuid,
+            ],
+            "enable_auto_applied_subscriptions_with_universal_link": False,
+            "has_custom_license_expiration_messaging": False,
+            "modal_header_text": None,
+            "expired_subscription_modal_messaging": None,
+            "button_label_in_modal": None,
+            "url_for_button_in_modal": None
+        }
+        self.mock_learner_license_result_response = {
+            "uuid": self.mock_learner_license_uuid,
+            "status": LicenseStatuses.ACTIVATED,
+            "user_email": self.mock_user_email,
+            "activation_date": _days_from_now(-10, DATE_FORMAT_ISO_8601_MS),
+            "last_remind_date": _days_from_now(-5, DATE_FORMAT_ISO_8601_MS),
+            "subscription_plan_uuid": self.mock_subscription_plan_uuid,
+            "revoked_date": None,
+            "activation_key": self.mock_license_activation_key,
+            "subscription_plan": self.mock_subscription_plan
+        }
+        self.mock_learner_license_activation_response = {
+            "uuid": self.mock_learner_license_activation_uuid,
+            "status": LicenseStatuses.ACTIVATED,
+            "user_email": self.mock_user_email,
+            "activation_date": _days_from_now(-10, DATE_FORMAT_ISO_8601_MS),
+            "last_remind_date": _days_from_now(-5, DATE_FORMAT_ISO_8601_MS),
+            "subscription_plan_title": "mock_subscription_plan_title",
+            "subscription_plan_expiration_date": _days_from_now(50, DATE_FORMAT_ISO_8601),
+            "activation_link": self.faker.url()
```

Comment:
Are these fields actually returned by the `license-activation` API? I believe it uses `LicenseSerializer`, which has the following fields ([source](https://github.com/openedx/license-manager/blob/2f0c9f40969aa1994ef001f56d21e8b6a4e0a6ae/license_manager/apps/api/serializers.py#L462-L480)):

```python
fields = [
            'uuid',
            'status',
            'user_email',
            'activation_date',
            'last_remind_date',
            'subscription_plan_uuid',
            'revoked_date',
            'activation_key',
        ]
```

However, this mock attribute is including `subscription_plan_title`, `subscription_plan_expiration_date`, and `activation_link` 🤔